dotcom-rendering/src/layouts/FrontLayout.tsx

```diff
@@ -81,20 +84,21 @@ const isToggleable = (
 export const decideAdSlot = (
 	renderAds: boolean,
 	index: number,
-	isNetworkFront: boolean | undefined,
 	collectionCount: number,
 	isPaidContent: boolean | undefined,
 	mobileAdPositions: (number | undefined)[],
 	hasPageSkin: boolean,
-	isInFrontsBannerTest?: boolean,
+	isInNetworkFrontsBannerTest?: boolean,
+	isInSectionFrontsBannerTest?: boolean,
 ) => {
 	if (!renderAds) return null;
 
 	const minContainers = isPaidContent ? 1 : 2;
 	if (
 		collectionCount > minContainers &&
 		index === getMerchHighPosition(collectionCount) &&
-		!(isInFrontsBannerTest && isNetworkFront)
+		!isInNetworkFrontsBannerTest &&
+		!isInSectionFrontsBannerTest
 	) {
 		return (
 			<AdSlot
```

Comment:
Users in the test but not on a front that is in the test, will not get the `merchandising-high` slots. I think we need the check to be something like:

`!(isInNetworkFrontsBannerTest && isFrontInTheNetworkTest) &&`
`!(isInSectionFrontsBannerTest && isFrontInTheSectionTest)`